packages/utils/src/cache/distributed-cache.ts

```diff
@@ -0,0 +1,307 @@
+import { Logger } from './helpers.js';
+import { Mutex } from 'async-mutex';
+import { CoreCache } from './core-cache.js';
+import { CacheMessenger } from './messenger.js';
+import type { CacheConfig } from './core-cache.js';
```

Comment:
```suggestion
import type { CacheConfig } from './core-cache';
```

Reply:
Updated, I change it to inline type import instead.